common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -3756,6 +3760,26 @@ object CelebornConf extends Logging {
       .checkValues(Set(ShuffleMode.HASH.name, ShuffleMode.SORT.name))
       .createWithDefault(ShuffleMode.HASH.name)
 
+  val CLIENT_PUSH_WRITER_MODE_BY_PARTITION_COUNT_THRESHOLD: ConfigEntry[Int] =
+    buildConf("celeborn.client.spark.push.writer.modeByPartitionCount.threshold")
+      .withAlternative("celeborn.push.writer.modeByPartitionCount.threshold")
+      .categories("client")
+      .doc("Threshold of shuffle partition count for push writer mode by partition count.")
+      .version("0.3.0")
```

Comment:
`0.3.0` has been released, it may be `0.3.2` OR `0.4.0`.

Reply:
Thanks, updated